### ISSUE TITLE
Was working, now get TypeError: Cannot read property 'firstChild' of null on line 1

### ISSUE THREAD

**Issue by brunodd** [skulpt/skulpt#360]
Everything worked about 5 days ago (last test was march 19th). Now when I try to run turtles, I get the following error:
~~~
TypeError: Cannot read property 'firstChild' of null on line 1
~~~

I used the following location for adding skulpt:
~~~
<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.6.2/jquery.min.js" type="text/javascript"></script>
<script src="http://www.skulpt.org/static/skulpt.min.js" type="text/javascript"></script> 
<script src="http://www.skulpt.org/static/skulpt-stdlib.js" type="text/javascript"></script> 
~~~

What is going on?

**Comment by albertjan** [comment]
@bnmnetp did a new release 2 days ago. :smile: Sorry this broke things for you. But @bnmnetp and @bzwheeler have been doing a lot of changes to the turtle module. Could you give us a bit more information? Preferably the bit of html you render the turtles too and the code where you initialize it.

You could for now install the previous version using [bower](http://bower.io) like this `bower install skulpt#0.9.8`.

**Comment by dsblank** [comment]
I'm seeing that too. I thought maybe it was on my end, as this is my first use of Skulpt from git.

**Comment by bnmnetp** [comment]
Everyone,

I'm sorry about the trouble.  We've been discussing this code for months now.  So I neglected to write up what I should have about the release.  Give me a few minutes to update the example gist and hopefully all will be back to ormal for you with a few easy updates to your html.

Brad

**Comment by bnmnetp** [comment]
OK, the gist now gives you a working example.  I wrote the following commentary on the github gist page, but I'll reproduce them here:

1.  Notice that we now wrap the ``Sk.importMainWithBody`` inside  ``Sk.misceval.asyncToPromise``  (lines 36-44)  This has the amazing effect that turtle now runs asynchronously!  No more will the turtle slow down as the drawing becomes larger, no more do precompute the entire drawing and then animate it later.  Now it runs in real time.  Which also means that we can, and do, support events:  onclick, onkey, ontimer!

2.  We also made the change that you no longer need to supply a canvas element. (line 61)  You now provide a div. and turtle will create the appropriate drawing context.  Still a canvas for now, but could be SVG or webgl or whatever in the future.

3.  Instead of Sk.canvas we now use the more generic way of connecting the canvas with ``(Sk.TurtleGraphics || (Sk.TurtleGraphics = {})).target = 'mycanvas';``  (line 35)

Again, apologies for making this such a rocky release.  The improvements are very well worth the trouble though!  And the actual turtle module has been in use by the guys at trinket.io and on my interactivepython.org website for some time.  So I'm confident that part of it is stable even if my communication skills are lacking.

Brad
